src/main/java/no/sikt/sws/models/opensearch/FailureDto.java

```diff
@@ -0,0 +1,45 @@
+package no.sikt.sws.models.opensearch;
+
+import static no.sikt.sws.constants.ApplicationConstants.EMPTY_STRING;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class FailureDto extends Dto {
+    @JsonProperty("shard")
+    public Number shard;
+
+    @JsonProperty("index")
+    public String index;
+
+    @JsonProperty("node")
+    public String node;
+
+    @JsonProperty("reason")
+    public ReasonDto reason;
+
+    public FailureDto() {
+        super();
+    }
+
+    @Override
+    public FailureDto stripper(String workspacePrefix) {
+        index = index.replaceAll(workspacePrefix + "-", EMPTY_STRING);
```

Comment:
stripper() is a core mechanic in SWS. It ensures that the user of SWS (nva-test) does not see the actual indexname. The actual index name is "workspace-nvatest-resources", while nva-test should belive its called "resources"